t/kubernetes/discovery/kubernetes.t

```diff
@@ -286,3 +294,5 @@ GET /compare
 Content-type: application/json
 --- response_body
 true
+--- error_log
+failed to get lua_shared_dict
```

Comment:
This test case is only to compare two configuration, not to test real k8s discovery functioni. I think it's ok to ignore the side-effect triggered by invalid configuration content.

Reply:
But providing a correct conf would be better, instead of using a wrong conf and ignore the error.

Reply:
Cannot. Because the shared_dict directives for k8s discovery is generated in normal APISIX startup, but not APISIX.pm.
This test case is only for configuration comparison, not to use the configuration.

Reply:
We can use `--- http_config` to provide such configuration if we can't generate them automatically.

Reply:
This test file is only used to compare if the configuration is set as expected, it does not care about whether the k8s function works or not, and in fact, the configuration is invalid and needs to get other auxiliary settings, but that is done by kubernetes2.t and kubernetes3.t. You have to repeat everything from those test files into this test file to shut up the errors, but it's meaningless because it's not the goal of this test file.
So all in all, no need to be sensitive to the error log here, as long as the response is ok.
So now I choose to add `--- ignore_error_log` here.